### PR TITLE
Add service metadata file

### PR DESCRIPTION
**Pull request by RedbackThomson**
Adds a `metadata.yaml` file to the root of the repository to describe the service's metadata and versions.

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+service:
+  full_name: "Amazon API Gateway"
+  short_name: "APIGWv2"
+  link: "https://aws.amazon.com/api-gateway/"
+  documentation: "https://docs.aws.amazon.com/apigateway/latest/developerguide/welcome.html"
```
Comment (A-Hilaly) [inline metadata.yaml:5]: shouldn't this be pointing to api gateway "v2" instead? /cc @vijtrip2

Comment (vijtrip2) [reply metadata.yaml:5]: no the service and docs is still "apigateway". apigatewayv2 is just subservice used for a certain product of apigateway.

Comment (A-Hilaly) [reply metadata.yaml:5]: Ohhh gotcha! thanks!

**Pull request merged**